modules/05_filter_calls.nf

```diff
@@ -2,6 +2,7 @@ params.memory_filter = "16g"
 params.cpus_filter = 2
 params.output = 'output'
 params.reference = false
+params.args_filter = "--min-allele-fraction 0.05 --min-reads-per-strand 1 --unique-alt-read-count 4"
```

Comment:
Let's make the default to have no parameters `""`. 
There are two reasons for this: 1) backwards compatibility of results with default parameters and 2) if you want to not use any filter you would need to do something like `--args_filter ""`, but I am not even sure if that would work and in any case it would not be easy to understand for a user.